wrapper/index.html

```diff
@@ -10,7 +10,8 @@
   <link href='https://fonts.googleapis.com/css?family=Ubuntu:400,500' rel='stylesheet' type='text/css'>
   <link rel="stylesheet" href="plugin.scss">
   <script src="https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.2.21/angular.min.js"></script>
-  <script src="../vendor/ui-bootstrap.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/angular-local-storage/0.1.4/angular-local-storage.js"></script>
+  <script src="../vendor/ui-bootstrap-tpls.min.js"></script>
```

Comment:
@tehpsalmist the wrong file was being included so i just updated to correct file - but do we want the minified version or not? I think not, because parcel can hopefully minify it ... (once we figure out that minify issue)

Reply:
right-o, we should probably get the non-minified.